db_stress_tool/db_stress_gflags.cc

```diff
@@ -598,6 +598,8 @@ DEFINE_bool(mock_direct_io, false,
 
 DEFINE_bool(statistics, false, "Create database statistics");
 
+DEFINE_string(statistics_string, "", "Serialized statistics string");
```

Comment:
Description is a little confusing. This is for creation of a 3rd party statistics object right?

Reply:
Yes. Let me update the description